docs/product/performance/metrics.mdx

```diff
@@ -99,28 +99,25 @@ For each project, you can configure how [Apdex](#apdex) and [User Misery](#user-
 
 The calculation method determines if duration is defined as the entire length of the transaction or as a specific [Web Vital](/product/performance/web-vitals/) such as LCP. The response time threshold determines what the satisfactory baseline duration is in milliseconds. This threshold may vary across projects depending on how user-facing a project is.
 
-## Custom Performance Metrics
+## Custom Performance Measurements
 
-In addition to the automatic performance metrics described above, Sentry supports setting custom performance metrics on transactions. Custom performance metrics allow you to define metrics (beyond the ones mentioned above) that are important to your application and send them to Sentry.
+<Include name="custom-measurements-disclaimer.mdx" />
 
-For example, you might want to set a custom metric to track:
+In addition to the automatic performance metrics described above, Sentry supports setting custom performance measurements on transactions. Custom performance measurements allow you to define measurements (beyond the ones mentioned above) that are important to your application and send them to Sentry.
+
+For example, you might want to set a custom measurement to track:
 
 - Total memory usage during a transaction
 - The amount of time being queried
 - Number of times a user performed an action during a transaction
 
-You define and configure custom metrics in the SDK. Currently, you can set custom metrics in:
+You define and configure custom measurements in the SDK. Currently, you can set custom measurements in:
 
-<Include name="custom-metrics-supported-sdks.mdx" />
+<Include name="custom-measurements-supported-sdks.mdx" />
 
-Custom metrics are accessible to you in [sentry.io](https://sentry.io) when you:
+Custom measurements are accessible to you in [sentry.io](https://sentry.io) when you:
 
 - Create visualizations in **Dashboards**
 - Query and add columns for them in **Discover**
-- View a transaction event detail (see image below)
```

Comment:
Are they viewable anywhere else right now after removing the event details view?